test/unit/pages/install/controller-test.js

```diff
@@ -220,4 +220,71 @@ describe('Install controller', function() {
     expect(InstallableItem.prototype.restartDownload).calledTwice;
     expect(installCtrl.closeDownloadAgainDialog).calledOnce;
   });
+
+  describe('checking the key for productname, productversion, productdesc, current, lable, show and status', function() {
+    let scopeStub = {
+      $apply: function(callback) {
+      callback && callback();
+      }
+    };
+
+    let timeoutStub = function(callback) {
+      callback && callback();
+    };
+    it('productName', function() {
+      sandbox.stub(InstallableItem.prototype, 'downloadInstaller').callsArgWith(2, 'timed out');
+      sandbox.stub(InstallableItem.prototype, 'restartDownload').returns();
+
+      let installCtrl = new InstallController(scopeStub, timeoutStub, installerDataSvc);
+      installCtrl.productName('vagrant');
```

Comment:
Every test should have expect() call in the end to verify everything worked as expected. In this example you should verify called method returns expected value.

``` JavaScript
expect(installCtrl.productName('vagrant')).to.equal('');
```